public/components/datasources/components/new/review_prometheus_datasource_configuration.tsx

```diff
@@ -84,7 +86,9 @@ export const ReviewPrometheusDatasource = (props: ConfigurePrometheusDatasourceP
               <EuiFlexItem grow={false}>
                 <EuiText className="overview-title">Authentication method</EuiText>
                 <EuiText size="s" className="overview-content">
-                  {currentUsername ? 'Basic auth' : 'AWSSigV4'}
+                  {currentAuthMethod === 'basicauth'
+                    ? 'Basic authentication'
+                    : 'AWS Signature Version 4'}
```

Comment:
No handling for `noauth`. Seems pervasive in some other spots as well.

Reply:
Prometheus only supports basic and sigv4